Jamulus.pro

```diff
@@ -295,12 +295,24 @@ win32 {
     isEmpty(PREFIX) {
         PREFIX = /usr/local
     }
+
     isEmpty(BINDIR) {
         BINDIR = bin
     }
     BINDIR = $$absolute_path($$BINDIR, $$PREFIX)
-    INSTALLS += target
     target.path = $$BINDIR
+    INSTALLS += target
+
+    isEmpty(APPSDIR) {
+        APPSDIR = share/applications
+    }
+    APPSDIR = $$absolute_path($$APPSDIR, $$PREFIX)
+    desktop.path = $$APPSDIR
+    desktop.files = distributions/jamulus.desktop
+    # the .desktop file assumes the binary is called jamulus
+    contains(CONFIG, "noupcasename") {
+        INSTALLS += desktop
+    }
 }
```

Comment:
We probably also need to install the icon, see https://github.com/corrados/jamulus/pull/247#discussion_r428560108.